src/NServiceBus.AcceptanceTests/Routing/When_publishing_to_scaled_out_subscribers_on_multicast_transports.cs

```diff
@@ -0,0 +1,86 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.ScenarioDescriptors;
+    using NUnit.Framework;
+
+    public class When_publishing_to_scaled_out_subscribers_on_multicast_transports : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Each_event_should_be_delivered_to_single_instance_of_each_subscriber()
+        {
+            await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b => b.When(c => c.EndpointsStarted, async (session, c) => { await session.Publish(new MyEvent()); }))
```

Comment:
@timbussmann The ATT framework sets the `EndpointsStarted` flag after all endpoints are successfully started